File: app/src/main/java/com/example/msimple/data/model/UserModel.java
package com.example.msimple.data.model;

import com.example.msimple.Delegate.LoginDelegate;
import com.example.msimple.data.vos.LoginUserVO;


public interface UserModel {

    void login(String emailOrPassword, String password, LoginDelegate loginDelegate);

    LoginUserVO getLoginUser();

}

File: app/src/main/java/com/example/msimple/data/model/UserModel.kt
package com.example.msimple.data.model

import com.example.msimple.delegate.LoginDelegate
import com.example.msimple.data.vos.LoginUserVO


interface UserModel {

    val loginUser: LoginUserVO

    fun login(emailOrPassword: String, password: String, loginDelegate: LoginDelegate)

}
